lib/Mojo/Server.pm

```diff
@@ -130,7 +132,17 @@ Application this server handles, defaults to a L<Mojo::HelloWorld> object.
   my $bool = $server->reverse_proxy;
   $server  = $server->reverse_proxy($bool);
 
-This server operates behind a reverse proxy, defaults to the value of the C<MOJO_REVERSE_PROXY> environment variable.
+This server operates behind a reverse proxy, defaults to the value of the C<MOJO_REVERSE_PROXY> environment variable
+or true if L</trusted_proxies> is not empty.
+
+=head2 trusted_proxies
+
+  my $proxies = $server->trusted_proxies;
+  $server     = $server->trusted_proxies(['10.0/8', '127.0/8', '172.16.0/12', '192.168.0/16']);
```

Comment:
Shouldn't the examples be consistent?

Reply:
oh, yes.

Sidenote: i had originally included these 4 original cidrs are they are the private network space defined for these purposes. I'm changing 128.0/8 to 127.0.0.1 because it is probably the most common usage of a private space and I rarely if ever see anyone use anything but .1